plugins/time-resources/src/components/EditToDo.svelte

```diff
@@ -235,9 +235,8 @@
   }
   .slots-content {
     gap: var(--spacing-2);
-    padding: var(--spacing-3) var(--spacing-4);
+    padding: var(--spacing-2) var(--spacing-4);
     border-top: 1px solid var(--theme-divider-color);
```

Comment:
Should we use border color from `lumia-colors`?

Reply:
I didn't use lumia colors for borders since we haven't changed the background colors yet